config/runtimes/vllm-multinode-template.yaml

```diff
@@ -0,0 +1,259 @@
+                  
+                  # Check model health
+                  curl --silent --max-time 5 --fail-with-body http://localhost:8080/health
+          startupProbe:
+            failureThreshold: 40
+            periodSeconds: 30
+            successThreshold: 1
+            timeoutSeconds: 5
+            exec:
+              command:
+                - bash
+                - -c
+                - |
+                  ray status
+
+                  # Check model health
+                  curl --silent --max-time 5 --fail-with-body http://localhost:8080/health
+          ports:
+            - containerPort: 8080
+              name: http
+              protocol: TCP
+      volumes:
+        - name: shm
+          emptyDir:
+            medium: Memory
+            sizeLimit: 12Gi    
+      workerSpec:
+        pipelineParallelSize: 2
+        tensorParallelSize: 1
+        containers:
+          - name: worker-container
+            image: $(vllm-image)
+            command: ["bash", "-c"]
+            args:
+              - |
+                SECONDS=0
+
+                while true; do              
+                  if (( SECONDS <= 120 )); then
+                    if ray health-check --address "${HEAD_SVC}.${POD_NAMESPACE}.svc.cluster.local:6379" > /dev/null 2>&1; then
+                      echo "Global Control Service(GCS) is ready."
+                      break
+                    fi
+                    echo "$SECONDS seconds elapsed: Waiting for Global Control Service(GCS) to be ready."
+                  else
+                    if ray health-check --address "${HEAD_SVC}.${POD_NAMESPACE}.svc.cluster.local:6379"; then
+                      echo "Global Control Service(GCS) is ready. Any error messages above can be safely ignored."
+                      break
+                    fi
+                    echo "$SECONDS seconds elapsed: Still waiting for Global Control Service(GCS) to be ready."
+                    echo "For troubleshooting, refer to the FAQ at https://docs.ray.io/en/master/cluster/kubernetes/troubleshooting/troubleshooting.html#kuberay-troubleshootin-guides"
+                  fi
+                  
+                  sleep 5
+                done
+
+                echo "Attempting to connect to Ray cluster at $RAY_HEAD_ADDRESS ..."
+                RAY_HEAD_ADDRESS="${HEAD_SVC}.${POD_NAMESPACE}.svc.cluster.local:6379"
+                ray start --address="$RAY_HEAD_ADDRESS" --block
+            env:
+              - name: POD_NAME
+                valueFrom:
+                  fieldRef:
+                    fieldPath: metadata.name
+              - name: POD_NAMESPACE
+                valueFrom:
+                  fieldRef:
+                    fieldPath: metadata.namespace
+            resources:
+              limits:
+                cpu: "16"
+                memory: 48Gi
+                nvidia.com/gpu: "1"
+              requests:
+                cpu: "8"
+                nvidia.com/gpu: "1"
```

Comment:
Same question as above here regarding the cpu/memory resources/requests

Reply:
my answer would be the same too here. Upstream also have a lot of discussion on this as well but it turned out it really depends on user situation. For user who want to just test, this looks very high but for user who want to deploy a really big model, it is not enough. So it should be configurable by user with InferenceService.
However I think gpu part should be removed for now because it is not overwritten by inferenceservice. it will be added if the gpu type is different.

Reply:
I removed the `nvidia.com/gpu`  so user must set this by isvc or they have to edit servingruntime. If they don't set , it will fail.

Reply:
> gpu part should be removed for now because it is not overwritten by inferenceservice.  It will be added if the gpu type is different.

Is this a bug? 

If that's the case, then it makes sense to remove GPU specifications here.

Reply:
The logic to merge/patch resource spec is here: https://github.com/kserve/kserve/blob/master/pkg/controller/v1beta1/inferenceservice/utils/utils.go#L403